views/index.ejs

```diff
@@ -2,14 +2,14 @@
 
 <html>
   <head>
+    <base href="<%= htmlWebpackPlugin.options.baseUrl %>" />
```

Comment:
Duplicated `<base />` tag. I suggest we drop all this extra `baseUrl` and `useBasePath` stuff for now. I think it's worth fixing, but this doesn't do it and this PR is the wrong place for it anyway.